eth/catalyst/api.go

```diff
@@ -474,6 +474,11 @@ func (api *ConsensusAPI) forkchoiceUpdated(update engine.ForkchoiceStateV1, payl
 				TxListHash:   &txListHash,
 			}
 			id := args.Id()
+			// If we already are busy generating this work, then we do not need
+			// to start a second process.
+			if api.localBlocks.has(id) {
```

Comment:
Is this `localBlocks` stored in memory?

Reply:
Yeah, the list is just stored in memory.